Assets/_Project/Scripts/UI/VariableController.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class VariableController : MonoBehaviour
```

Comment:
Does this class just act as a setter for `ParameterPass`?  If so, is there a reason we don't just add public setters to `ParameterPass`?  Also, if this class needs to exist for whatever reason it could again use a much more descriptive name.